plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -0,0 +1,185 @@
+package io.jenkins.plugins.casc;
+
+import io.jenkins.plugins.casc.impl.configurators.HeteroDescribableConfigurator;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+public class SchemaGeneration {
```

Comment:
`@Restricted(NoExternalUse.class)` ?

Reply:
https://github.com/jenkinsci/configuration-as-code-plugin/issues/1028 FTR

Reply:
I made the changes :+1:

Reply:
You did not. Maybe `@Restricted(Beta.class)` is even better